### PR TITLE
chore: fix ESLint warnings

### PR DESCRIPTION
**Pull request by MichaelDeBoey**

### CODE REVIEW

**Comment by MichaelDeBoey** [comment]
Since we already have the plugin in our dependencies & it's an official one from the Cypress team itself, I think enabling it would be a good idea, just like we do for Testing Library in `@remix-run/eslint-config/jest-testing-library`

File: app/routes/notes/$noteId.tsx [hunk 1]
```diff
@@ -3,8 +3,7 @@ import { json, redirect } from "@remix-run/node";
 import { Form, useCatch, useLoaderData } from "@remix-run/react";
 import invariant from "tiny-invariant";
 
-import { deleteNote } from "~/models/note.server";
-import { getNote } from "~/models/note.server";
+import { deleteNote, getNote } from "~/models/note.server";
```
Comment (MichaelDeBoey) [inline app/routes/notes/$noteId.tsx:6]: This has nothing to do with enabling the rule, ESLint was just warning for this, so I fixed it

File: .eslintrc.js [hunk 2]
```diff
@@ -17,7 +16,7 @@ module.exports = {
   // set the jest version.
   settings: {
     jest: {
-      version: 27,
+      version: 28,
```
Comment (MichaelDeBoey) [inline .eslintrc.js:19]: Figured I'd update to latest version as well

**Comment by kentcdodds** [comment]
Right, but we control the rules enabled in our own config and we only enable the rules that are legitimately helping avoid issues. I don't know about the default cypress config. Even in this PR we have to disable a rule for some of the code we've got. At Remix in particular, we're allergic to rules like that one. Only those which *actually* cause bugs should be enabled.

**Comment by MichaelDeBoey** [comment]
So I'll remove the config & just do the other changes from this PR?

**Comment by kentcdodds** [comment]
Sounds good 👍

**Comment by MichaelDeBoey** [comment]
@kentcdodds Should be fine now

**Review by kentcdodds — APPROVED** [review]
Super, thanks!

**Pull request merged**